Mathlib/LinearAlgebra/TensorProduct/LinearMap.lean

```diff
@@ -0,0 +1,86 @@
+/-
+Copyright (c) 2024 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
```

Comment:
I'd argue this should be
```suggestion
Copyright (c) 2023 Eric Wieser. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Eric Wieser, Christopher Hoskin
```
since I think this is largely copied from the other file.

Reply:
Fair enough!